### DISCUSSION TITLE
Lesson 3 : FundMe / safeMath

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#685] [Q&A] [answered]
Hello, 

If solidity >=0.8.0 
we souldn't import SafeMathChainLink.sol ? 

Because i don't find the link to copy paste the url to Import safeMath on the contract. 

I just find a GitHub link. 
not an example like import "@chalinlink/contracts/....../SafeMathChainLink.sol" 
like the video tutorial.

Or like this [page](https://docs.chain.link/docs/get-the-latest-price/) for AggregatorV3Interface.sol

**Comment by cromewar** [comment, marked as answer]
Hello @NinjaTurtles-cloud  Safe Math is completely obsolete on Solidity version 0.8.0 so you can drop it safely.
